### ISSUE TITLE
Keyboard shortcut to speak a user's state

### ISSUE THREAD

**Issue by bear101** [BearWare/TeamTalk5#36]
The user who is talking is shown in green but screen-readers cannot detect this, so there should be a keyboard shortcut to speak out a user's state.

**Comment by bear101** [comment]
Ctrl+G to speak user and channel info.
Fixed in 5.0.0.3947.

**Comment by beqabeqa473** [comment]
hi. did you pushed it to git repo?

**Comment by bear101** [comment]
It is now
